neofetch

```diff
@@ -426,6 +435,12 @@ get_uptime() {
                     h="${h#0}" t="${t#0}"
                     seconds="$((d*86400 + h*3600 + ${t%%:*}*60 + ${t#*:}))"
                 ;;
+
+                "FreeMiNT")
```

Comment:
Can this not be merged with the BSD/MacOS uptime detection?

Reply:
I know the outputs of the same command are different but the regex matching should work on both outputs.